src/sys/DISK.ASM

```diff
@@ -121,13 +121,20 @@ get_file_info:                                  ; Get an item (AL=offset) from a
 
 
 ; LOAD FILE FROM CLUSTER NUMBER: Loads file starting from the cluster number given in memory into the file buffer
-; IN: [cluster] = Starting cluster number, [bytes_to_load] = File size (run get_file_cluster to set these up), [file_buffer_pointer] = Offset to load to
-; OUT: CF = Set on error
+; IN: [root_entry] = Root entry number of the file; [file_buffer_pointer] = Offset to load to
+; OUT: [file_buffer_pointer] = Initial value + bytes copied; CF = Set on error
```

Comment:
With the incorporation of device files, the size of the file cannot be determined through retrieval from the directory entry.